index.bs

```diff
@@ -736,6 +736,191 @@ Depending on the underlying platform, the user agent <span class=allow-2119>may<
 
 For a history and rationale of this design, please see the <a href="https://github.com/webmachinelearning/webnn/blob/master/device-selection-explainer.md">device selection explainer</a>.
 
+## Operators ## {#programming-model-operators}
+
+*This section is non-normative.*
+
+The WebNN API defines a set of operators required by well-known CNN and RNN, transformer and generative models that address key [[#usecases-application]]. The details of each operator are defined in the normative sections of this specification, in alphabetical order by the operator name. These operators are grouped into categories based on their functionality in the following non-normative table to give a functional overview of the API surface.
+
+Note: Some operators belong to multiple categories. For example, {{MLGraphBuilder/clamp()}} is both a math function and also used as an activation.
+
+<table id='operators-table' class='data'>
+  <caption>Operators by category</caption>
+  <thead>
+    <tr>
+      <th>Category</th>
+      <th>Operators​</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Tensor creation</td>
+      <td>
+      {{MLGraphBuilder/input()}},
+      {{MLGraphBuilder/constant(tensor)|constant()}}
+      </td>
+    </tr>
+    <tr>
+      <td>Tensor rearrangement</td>
+      <td>
+      {{MLGraphBuilder/concat()}},
+      {{MLGraphBuilder/expand()}},
+      {{MLGraphBuilder/gather()}},
+      {{MLGraphBuilder/gatherElements()}},
+      {{MLGraphBuilder/scatterElements()}},
+      {{MLGraphBuilder/gatherND()}},
+      {{MLGraphBuilder/scatterND()}},
+      {{MLGraphBuilder/where()}},
+      {{MLGraphBuilder/pad()}},
+      {{MLGraphBuilder/reshape()}},
+      {{MLGraphBuilder/slice()}},
+      {{MLGraphBuilder/split()}},
+      {{MLGraphBuilder/transpose()}},
+      {{MLGraphBuilder/resample2d()}},
```

Comment:
I feel `resample2d` is not a tensor rearrangement op, because it doesn't just rearrange the existing values, but generate new values. FYI, CoreML puts "resample" into "image resizing" category: https://apple.github.io/coremltools/source/coremltools.converters.mil.mil.ops.defs.html#coremltools.converters.mil.mil.ops.defs.iOS16.image_resizing.resample

Reply:
> but 1D resampling is useful for audio rescaling too (so I wouldn't limit it to just "image resizing")

Agreed. For 2D resmaple/resize, TOSA also places it under "image operators" category: https://www.mlplatform.org/tosa/tosa_spec.html#_image_operators